src/main/java/com/alkemy/ong/controllers/CommentController.java

```diff
@@ -106,7 +104,7 @@ public ResponseEntity<BasicCommentDto> editComment(
 		return ResponseEntity.ok(ObjectMapperUtils.map(comment, BasicCommentDto.class));
```

Comment:
Just as a suggestion, what if we make logic in the service and make controller as clean as it's possible?

Reply:
corrected